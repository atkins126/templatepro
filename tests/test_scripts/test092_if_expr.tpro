{{if value1|empty}}
1-THEN
{{else}}  
2-ELSE
{{endif}}

3-{{:value1|uppercase}}

{{if value1|uppercase}}
4-THEN
{{else}}  
5-ELSE
{{endif}}

{{if !value1|empty}}
6-THEN
{{else}}  
7-ELSE
{{endif}}

***

{{for objb in objectsb}}
**{{:objb.@@index}}
Prop1 = {{:objb.Prop1}}
{{if objb.Prop1|uppercase }}
Prop1 is TRUE
{{else}}
Prop1 is FALSE
{{endif}}
{{endfor}}