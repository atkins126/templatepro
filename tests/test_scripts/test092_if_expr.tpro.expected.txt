  
2-ELSE

3-TRUE

4-THEN

  
7-ELSE

***

**1
Prop1 = true
Prop1 is TRUE
**2
Prop1 = false
Prop1 is FALSE
**3
Prop1 = 1
Prop1 is TRUE
